packages/Input/__tests__/Input.spec.jsx

```diff
@@ -69,6 +69,20 @@ describe('Input', () => {
 
       expect(label).toContainReact(<Text size="small">The short hint</Text>)
     })
+
+    it('can have a long hint', () => {
+      const { input } = doMount({
+        id: 'the-id',
+        hint: 'The long hint over here',
+        hintPosition: 'below',
+      })
+
+      expect(input).toContainReact(
+        <Paragraph id="the-id_hint" size="medium">
+          The long hint over here
```

Comment:
Consider using an example of a hint that is so long that it can't be used as an inline hint.

Reply:
Will find 85 characters of lorem ipsum!

Reply:
You're right that this doesn't make any difference while programmatic testing, so we can safely skip this part.